codex/rest/api.nim

```diff
@@ -45,6 +46,33 @@ proc validate(
   {.gcsafe, raises: [Defect].} =
   0
 
+proc formatEnginePeers(peers: OrderedTable[PeerId, BlockExcPeerCtx]): JsonNode =
+  let jarray = newJArray()
+  for key, value in peers:
+    jarray.add(%*
+      {
+        "peerId": $key,
+        "context": {
+          "blocks": value.blocks.len,
+          "peerWants": value.peerWants.len,
+          "exchanged": value.exchanged,
+          "lastExchange": $(value.lastExchange),
+        }
+      })
+
+  return jarray
+
+proc formatSwitchPeers(peers: Table[PeerId, seq[MultiAddress]]): JsonNode =
+  let jarray = newJArray()
+  for key, value in peers:
+    jarray.add(%*
+      {
+        "peerId": $key,
+        "numberOfAddresses": $value.len
+      })
+
+  return jarray
+
```

Comment:
formatEnginePeers and formatSwitchPeers can be remove, as well as their use in the JSON response below.